src/analyze/analyze-verify-util.c

```diff
@@ -99,6 +102,9 @@ int verify_set_unit_path(char **filenames) {
         if (!joined)
                 return -ENOMEM;
 
+        if (isempty(joined))
+                return 0;
```

Comment:
it's a bit unnecessary to first concat the entries in `ans` into a string, and then check if that is empty. I think it would make more sense to check `strv_isempty(ans)` first, and already give up then. But doesn't matter

Reply:
-> #30186.